src/test/url-handling.test.js

```diff
@@ -26,20 +26,25 @@ function _getStoreWithURL(
     pathname?: string,
     search?: string,
     hash?: string,
+    v?: number | false, // If v is false, do not add a v parameter to the search string.
```

Comment:
The issue i'm fixing here is that by default `_getStoreWithURL` would run all URL upgrade steps. To avoid that we'd have needed to specify `v=${CURRENT_URL_VERSION}` in all search strings.

Instead in this patch I default to always using `CURRENT_URL_VERSION` unless we specify a specific `v` or even `false` to omit it.

Reply:
Thanks for fixing this issue!